examples/axis-center/index.js

```diff
@@ -0,0 +1,56 @@
+const categories = [
+    { text: 'Fruit', domain: ['Apples', 'Bananas'] },
+    { text: 'Meat', domain: ['Sausages'] },
+    { text: 'Drink', domain: ['Soda'] },
+    { text: 'Vegetables', domain: ['Pickles', 'Aubergines'] }
+];
+
+const bandContainer = document.querySelector('#band-container');
+
+const domain = categories.reduce((acc, curr) => [...acc, ...curr.domain], []);
+
+const bandScale = d3.scaleBand().domain(domain);
+
+const bandAxis = fc.axisOrdinalBottom(bandScale).tickPadding(5);
+
+d3.select(bandContainer)
+    .on('draw', () => {
+        d3.select(bandContainer)
+            .select('svg')
+            .append('g')
```

Comment:
The `g` elements are redundant in this case. I think they'd only be required for these examples if there are multiple components being rendered.